src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -2108,6 +2109,17 @@ public void testNetworksConnectContainer() throws Exception {
     assertThat(network.containers().size(), equalTo(1));
     assertThat(network.containers().get(containerCreation.id()), notNullValue());
 
+    final ContainerInfo containerInfo = sut.inspectContainer(containerCreation.id());
+    assertThat(containerInfo.networkSettings().networks().size(), is(2));
+    final AttachedNetwork attachedNetwork =
+        containerInfo.networkSettings().networks().get(networkName);
+    assertThat(attachedNetwork, is(notNullValue()));
+    assertThat(attachedNetwork.endpointId(), is(notNullValue()));
+    assertThat(attachedNetwork.gateway(), is(notNullValue()));
+    assertThat(attachedNetwork.ipAddress(), is(notNullValue()));
+    assertThat(attachedNetwork.ipPrefixLen(), is(notNullValue()));
+    assertThat(attachedNetwork.macAddress(), is(notNullValue()));
```

Comment:
Can we also test `ipv6Gateway()`, `globalIPv6Address()`, and `globalIPv6PrefixLen()` here?